modules/ROOT/pages/dataweave-runtime-functions.adoc

```diff
@@ -41,6 +41,8 @@ cannot use the `Mule` namespace or use Mule Runtime functions in custom
 modules, only in DataWeave scripts and mappings.
 ====
 
+For DataWeave runtime properties, see the DataWeave functions xref:dw-runtime-functions-props.adoc[] and xref:dw-runtime-functions-prop.adoc[].
```

Comment:
Add the missing topic names in [], for example [Runtime Functions Property]. Also, is it possible that there is a typo in the xrefs since there are very similar, except that one is plural (props) and the other is singular (prop)

Reply:
@JennyHajee The topic names aren't needed.

Reply:
@JennyHajee There are 2 functions, one called prop and the other called props.